### ISSUE TITLE
Add `yardstick::specificity()` and `yardstick::sensitivity()` aliases

### ISSUE THREAD

**Issue by DavisVaughan** [tidymodels/yardstick#127]
For `spec()` and `sens()`. This would help avoid a collision with `readr::spec()` that looks like:

```
> class_metrics <- yardstick::metric_set(accuracy, ppv, sens, spec)
Error: 
The combination of metric functions must be:
- only numeric metrics
- a mix of class metrics and class probability metrics
The following metric function types are being mixed:
- class (accuracy, ppv, sens)
- other (spec)
Call `rlang::last_error()` to see a backtrace.
```

**Comment by topepo** [comment]
This would also be good since they are the cases where the statistic name is abbreviated (accuracy is `accuracy` but sensitivity is just `spec`). @apresshill

**Comment by DavisVaughan** [comment]
It would be fine to just do something like:

```r
#' @rdname sens
#' @export
sensitivity <- sens
```

The idea is to export `sensitivity()` using the same function as `sens()`, and then document it with `sens()`. Same for `spec()`

**Comment by DavisVaughan** [comment]
Closed by #136
